.php_cs.dist

```diff
@@ -52,6 +52,8 @@ return PhpCsFixer\Config::create()
         'nullable_type_declaration_for_default_null_value' => [
             'use_nullable_type_declaration' => false,
         ],
+        'line_ending' => true,
```

Comment:
This is implied by the `@PSR2`, `@Symfony`, `@PhpCsFixer`, do not mess these configs with unneeded rules. I spend a lot of time to provide clean, strict, but still not too strict rules.